
# Materials and Methods

## Study area

The study area, described by Bowser et al. [-@Bowser_et_al_2020], was a portion of the Slikok Creek watershed on the Kenai National Wildlife Refuge (KNWR). The resulting 938 ha study area occupied a bounding box from 60.44° to 60.47° latitude and from -151.10° to -151.03° longitude.

Based on data from the National Land Cover Database [@Homer_et_al_2015], the land area of this study area was coverd mostly by mixed forest (29%), evergreen forest (22%), emergent herbaceous wetlands (13%), deciduous forest (12%), woody wetlands (11%), and open water (10%). Common trees in well-drained areas were white spruce (*Picea glauca* (Moench) Voss), Alaska birch (*Betula pendula* subsp. *mandshurica* (Regel) Ashburner & McAll.), quaking aspen (*Populus tremuloides* Michx.), and cottonwood (*Populus* × *hastata* Dode). Black spruce (*Picea mariana* Britton, Sterns & Poggenb.) was the dominant tree in forested wetlands. Herbaceous wetlands were characterized by mosses, sedges, and low shrubs. Shallow, eutrophic lakes (Headquarters Lake, Nordic Lake, and smaller, unnamed lakes) accounted for most of the open water in the study area.

Sampling design, field methods, and identification methods were detailed by @Bowser_et_al_2020, but they are summarized here.

## Sampling design

A grid with 500 m spacing between points was chosen by using the coordinates of the centroids of the 250 m pixels from the Alaska eMODIS product [@Jenkerson_2010], choosing every other centroid to make a grid of sites having 500 m spacing. The resulting sample frame consisted of 40 terrestrial sites.

## Field methods

Sampling sites were marked by driving 122 cm long, 13 mm diameter SunGUARD Smart Stake™ fibreglass rods into the ground, then labelling them with aluminium tags. During the survey period, sites were also temporarily marked with high-visibility forestry flagging tape.

Sweep net samples of terrestrial arthropods were collected from 14 to 17 June 2016. A second set of sweep net samples was collected from 18 July to 9 August 2016. A total of 160 sweep net samples were collected (40 plots × 2 samples/plot × 2 visits/plot).

Arthropods were sampled within a 100 m<sup>2</sup>, 5.64 m radius, circular plot using the center stake as plot center. To enable comparison with the previous work of @Morton_et_al_2009, we used the same methods except that we subsampled spatially. We split the plot into two subplots, dividing along the north-south axis. Each semicircular subplot was independently sweep-netted, such that the entire area was swept from the ground surface up to a height of roughly 2 m. No defined pattern of sweeping was enforced, but we ensured that all substrates and macrohabitats within reach were swept over once within a time limit of 5 min per sample. We used a BioQuip™ model 7112CP 30.5 cm diameter net with a BioQuip™ model 7312AA 30.5 cm extension handle and a BioQuip™ model 7112CPA net bag with a mesh size of approximately 8 × 9 meshes/mm.

All specimens were collected into a single Nalgene® model 2104-0008 wide-mouth 250 ml bottle containing UniGard -100 propylene glycol antifreeze.

## Laboratory methods

All invertebrates in the sweep net samples were separated from debris by hand under a stereomicroscope. All fragments of invertebrates were retained.

Due to budget limitations, we processed 125 of the 160 sweep net samples. We selected all 80 samples taken from the east side of each plot (40 plots × 1 sample/plot × 2 visits/plot). To choose 45 samples from the remaining 80, we selected plots spatially. First, we chose 20 samples from plots at 1 km spacing (10 plots × 2 visits/plot), then we chose 25 of 26 samples from another 13 plots that were maximally distant from these 10 plots (13 plots × 2 visits/plot). These 45 samples from west plot halves were intended to be used for estimating occupancy metrics.

Sweep net samples were shipped to RTL Genomics, Lubbock, Texas (<http://rtlgenomics.com>) for extraction and DNA sequencing steps. For details of sequencing and identifications obtained through a high-throughput sequencing analysis pipeline, see -@Bowser_et_al_2020.

Invertebrate sequences that could not be confidently assigned to described species were assigned to BOLD Barcode Index Numbers [BINs, @Ratnasingham_Hebert_2013] if possible. Sequences that could be assigned to neither species nor BINs were given provisional names including labels of the molecular operational taxonomic units [MOTUs, @Blaxter_et_al_2005], e.g. "*Liriomyza* sp. SlikokOtu253". For our purposes we considered all of these entities to be species.

We sought to follow the guidelines of @Penev_et_al_2017 for publication of biodiversity data. Species occurrence data have been made available via Arctos (<https://arctosdb.org/>), where they are associated together via an Arctos project (<http://arctos.database.museum/project/10002227>). These occurrence data on Arctos are also provided to the Global Biodiversity Information Facility (<https://www.gbif.org/>).

## Data analysis

In the time since @Bowser_et_al_2020 was published, a handfull of identifications of the records on Arctos have been improved based on new data that became available in the reference databases [@Ratnasingham_Hebert_2007; @Clark_et_al_2016].

A summary of identfications from all HTS occurrences from sweep net samples obtained by @Bowser_et_al_2020 were downloaded from Arctos on 12 November 2020 (saved search URI: <https://arctos.database.museum/saved/2020-11-12-1157_Slikok_metabarcoding_taxa_list>), yielding a list of 975 unique identifications. For unidentified MOTUs where 10 or more occurrences were observed, we attempted to improve identifications by submitting the sequences to BOLD's ID Engine [@Ratnasingham_Hebert_2007] and BLAST [@Altschul_et_al_1990] searches. We obtained no new identifications.

The 2,375 occurrences were downloaded on 12 November 2020 (saved search URI: <https://arctos.database.museum/saved/2020-11-12-1400_Slikok_project_metabarcoding_occurrences>).
